requirements/base.in

```diff
@@ -1,5 +1,10 @@
-unstructured[local-inference]>=0.7.1
+unstructured[local-inference]>=0.8.1
 unstructured-api-tools>=0.6.0
+pydantic<2.0.2
```

Comment:
pydantic-2.0.2 returns the following error when I make an API request as publicly documented:
```
{
  "detail": [
    {
      "type": "list_type",
      "loc": [
        "body",
        "files"
      ],
      "msg": "Input should be a valid list",
      "input": {
        "filename": "layout-parser-paper.pdf",
        "file": {
          "_file": {},
          "_max_size": 1048576,
          "_rolled": true
        },
        "size": 4686220,
        "headers": {
          "content-disposition": "form-data; name=\"files\"; filename=\"layout-parser-paper.pdf\"",
          "content-type": "application/pdf"
        }
      },
      "url": "https://errors.pydantic.dev/2.1.2/v/list_type"
    }
  ]
}

```